connection-limit/connection_limit.bpf.c

```diff
@@ -62,23 +62,26 @@ struct {
     __uint(type, BPF_MAP_TYPE_ARRAY);
     __uint(key_size, sizeof(uint32_t));
     __uint(value_size, sizeof(uint64_t));
-   __uint(max_entries, 1);
+    __uint(max_entries, 1);
+    __uint(pinning, LIBBPF_PIN_BY_NAME);
 } cl_max_conn SEC(".maps");
 
 /* Maintains TCP listen ports */
 struct {
     __uint(type, BPF_MAP_TYPE_HASH);
     __uint(key_size, sizeof(uint16_t));
     __uint(value_size, sizeof(uint32_t));
-   __uint(max_entries, 200);
+    __uint(max_entries, 200);
+    __uint(pinning, LIBBPF_PIN_BY_NAME);
 } cl_tcp_conns SEC(".maps");
 
 /* Maintains concurrent connection sockets */
 struct {
     __uint(type, BPF_MAP_TYPE_HASH);
     __uint(key_size, sizeof(uint64_t));
     __uint(value_size, sizeof(uint32_t));
-   __uint(max_entries, 30000);
+    __uint(max_entries, 30000);
```

Comment:
Can we use MACROs for the max_entries instead of hard-coded value for all the maps ?

Reply:
It's not repeating, what is the advantage of macro here?